pennylane_lightning/src/benchmarks/Bench_LinearAlgebra.cpp

```diff
@@ -111,18 +111,20 @@ template <class T> static void omp_innerProd_cmplx(benchmark::State &state) {
 
     for (auto _ : state) {
         std::complex<T> res(.0, .0);
+        // Create indirection to avoid GCC issue with AVX512 compilation
```

Comment:
Nice catch!

Reply:
The simplest fix for a very obtuse error

Reply:
+1